alarm/AlarmManager2.js

```diff
@@ -76,6 +76,8 @@ const alarmDetailPrefix = "_alarmDetail";
 
 const _ = require('lodash');
 
+const minimatch = require('minimatch');
+
```

Comment:
unused?

Reply:
Sorry, I forgot to delete the debugging code.